bundles/org.openhab.binding.solarman/src/main/java/org/openhab/binding/solarman/internal/util/ClassUtils.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.solarman.internal.util;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link StringUtils} common class utility functions
+ *
+ * @author Catalin Sanda - Initial contribution
+ */
+@NonNullByDefault
+public class ClassUtils {
+    public static String getShortClassName(@Nullable Class<?> cls) {
```

Comment:
Can you add javadoc to all util public methods ?
(this and also the other files)